build-constraints.yaml

```diff
@@ -5892,6 +5893,7 @@ packages:
         - slist # #5965/closed
         - syb-with-class
         - type-errors-pretty # #5965/closed
+        - unix-compat
```

Comment:
@andreasabel I'm not sure, this comment above "abandoned" seems to describe the situation accurately, though:

```
    # If you stop maintaining a package (either just on stackage, or
    # completely), you can move it here. It will be disabled if it
    # starts causing problems.
```

Reply:
is there anyone else in [@haskell-pkg-janitors](https://github.com/haskell-pkg-janitors) who would like to maintain this, or is the plan to abandon/deprecate the package?

Reply:
Well, if there is no one else, I can take the Stackage maintainership for `unix-compat`.